.rubocop_todo.yml

```diff
@@ -1,12 +1,67 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2019-04-19 14:01:54 -0400 using RuboCop version 0.67.2.
+# on 2021-09-21 15:07:21 UTC using RuboCop version 1.21.0.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
-# Offense count: 3
-Lint/IneffectiveAccessModifier:
+# Offense count: 6
+# Cop supports --auto-correct.
+# Configuration parameters: TreatCommentsAsGroupSeparators, ConsiderPunctuation, Include.
+# Include: **/*.gemfile, **/Gemfile, **/gems.rb
+Bundler/OrderedGems:
```

Comment:
_I think_ the various *gemfiles might not be violating the 👮‍♂️ s. Can we try removing all todos and see if it breaks things?

Reply:
Huh you're right - I did this:

```
rm .rubocop_todo.yml
bundle exec rubocop -A --auto-gen-config
```

And these parts of the file were removed - not quite sure how that happened. 🤔